### PR TITLE
CI: fix deny

### PR DESCRIPTION
**Pull request by Boog900**
I regenerated the file so we don't get depreciated warnings, I think I kept most options the same.

@hinto-janai

### CODE REVIEW

File: deny.toml [hunk 1]
```diff
@@ -64,35 +70,18 @@ feature-depth = 1
...
-# The lint level for security vulnerabilities
-vulnerability = "deny"
-# The lint level for unmaintained crates
-unmaintained = "warn"
-# The lint level for crates that have been yanked from their source registry
-yanked = "deny"
-# The lint level for crates with security notices. Note that as of
-# 2019-12-17 there are no security notice advisories in
-# https://github.com/rustsec/advisory-db
-notice = "warn"
```
Comment (hinto-janai) [inline deny.toml:80]: Removed accidentally? Are these the default when running `cargo deny` anyway?

File: deny.toml [hunk 2]
```diff
@@ -103,61 +92,37 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-# The lint level for crates which do not have a detectable license
-unlicensed = "deny"
```
Comment (hinto-janai) [inline deny.toml:107]: Keep this, no license basically means [we can't use it.](https://choosealicense.com/no-permission/)

File: deny.toml [hunk 3]
```diff
@@ -103,61 +92,37 @@ ignore = [
...
-# * fsf - The license will be approved if it is FSF Free
-# * osi-only - The license will be approved if it is OSI-approved *AND NOT* FSF
-# * fsf-only - The license will be approved if it is FSF *AND NOT* OSI-approved
-# * neither - This predicate is ignored and the default lint level is used
-allow-osi-fsf-free = "neither"
-# Lint level used when no other predicates are matched
-# 1. License isn't in the allow or deny lists
-# 2. License isn't copyleft
-# 3. License isn't OSI/FSF, or allow-osi-fsf-free = "neither"
-default = "deny"
```
Comment (hinto-janai) [inline deny.toml:160]: Removed accidentally?

File: deny.toml (hunk 1 above)
Comment (Boog900) [reply deny.toml:80]: individual setting has been depreciated, they are all deny now.

File: deny.toml (hunk 3 above)
Comment (Boog900) [reply deny.toml:160]: same reason here as previous comment

**Review by hinto-janai — APPROVED** [review]

**Pull request merged**